webpack.config.js

```diff
@@ -0,0 +1,179 @@
+const path = require("path");
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const ReactRefreshWebpackPlugin = require("@pmmmwh/react-refresh-webpack-plugin");
+const TerserPlugin = require("terser-webpack-plugin");
+const CopyPlugin = require("copy-webpack-plugin");
+const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
+const { createBabelConfig } = require('./babel.config.js');
+const { DefinePlugin, ProvidePlugin } = require("webpack");
+const JSONManifestPlugin = require("./jsonmanifest-plugin");
+
+const createStats = verbose => ({
+    assets: verbose,
+    builtAt: verbose,
+    cached: false,
+    children: false,
+    chunks: false,
+    colors: true,
+    entrypoints: true,
+    hash: false,
+    modules: false,
+    performance: false,
+    timings: verbose,
+    version: verbose,
+});
+
+module.exports = (env, argv) => ({
+    context: path.resolve(__dirname),
+    // Our application entry point.
+    entry: {
+        main: [
+            "./src",
+        ],
+    },
+    devtool: 'inline-source-map',
```

Comment:
I'd rather keep it seperate unless theres a good reason not to
```suggestion
    devtool: 'source-map',
```